### ISSUE TITLE
Sign-in notification for specific contact

### ISSUE THREAD

**Issue by a-iv** [redsolution/xabber-android#86]

**Comment by daviessm** [comment]
I am working on this issue. More details will appear here soon...

**Comment by grigoryfedorov** [comment]
@daviessm no.

**Comment by BlueMax** [comment]
@daviessm:
Are you still working on it? Would love to see this feature as well.

**Comment by daviessm** [comment]
@BlueMax It's been on hold for a long time while I worked on other things.

I initially came across a problem in that when an account signs in, all the contacts that are online are registered as coming online and there's no way to know when the initial sign-on is complete so to start notifying. Short of having, say, a five second timeout before starting notifications I'm not sure of the best way to handle that - although thinking about it, all the other XMPP clients must do something similar.
